MdePkg/Include/IndustryStandard/Nvme.h

```diff
@@ -984,6 +1005,30 @@ typedef enum {
   SanitizeOpcode                = NVME_ADMIN_SANITIZE_CMD
 } NVME_ADMIN_COMMAND_OPCODE;
 
+// MU_CHANGE [BEGIN] - Add Nvm Express Admin Feature Identifiers

```

Comment:
These are coming from Nvme specification.

They should probably be moved into Mde\Include\IndustryStandard\Nvme.h for public reuse.